### DISCUSSION TITLE
Persist progress when reviewing a pull request on GitHub

### DISCUSSION THREAD

**Discussion by Craig-von-Land** [community/community#45478] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

When reviewing files in a pull request, I can mark each file as "Viewed" when I'm done with it. That doesn't seem to persist if the browser is closed and I go back to the same review again later to finish.

What is the recommended way to persist review progress on a large pull request that I might not have time to complete in one sitting?

**Comment by jge162** [comment]
@Craig-von-Land you can use the "Review changes" feature to review the pull request and leave comments on specific lines of code. You can then save your progress by clicking the "Leave review" button, and come back to the review later by clicking the "Review changes" button again. This will allow you to pick up where you left off and continue reviewing the pull request.

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request

**Reply by Craig-von-Land** [reply]
So, basically leave temporary comments like bookmarks on the files that I have reviewed, and then go back and delete all those comments again before submitting the review to the pull request owner?

**Reply by jge162** [reply]
Yes @Craig-von-Land bookmark/comments can serve as a reminder of where you left off and which files you still need to review.

Once you are done with your review and before you submit it to the pull request owner, you can go back and delete all the temporary comments. This will keep the pull request clean and easy to read for the pull request owner. 👍
